test/admin/e2e/list-view/e2e.spec.ts

```diff
@@ -1283,6 +1302,39 @@ describe('List View', () => {
       await expect(page.locator('.row-2 .cell-number')).toHaveText('1')
     })
 
+    const openColumnControls = async () => {
+      await page.locator('.list-controls__toggle-columns').click()
+      await expect(page.locator('.list-controls__columns.rah-static--height-auto')).toBeVisible()
+    }
+
+    test('should allow sorting by nested field within group in separate column', async () => {
```

Comment:
We should add an explicit test in addition to this one that ensures the group columns are rendered top level. This test is specific for the sorting behavior.